### PR TITLE
Lesson 6

### PR DESCRIPTION
**Pull request by EvgeniyTsarkov**
The following tasks have been added:
Main Task 28
Alternative Task: Annotations Task
Please Check!

### CODE REVIEW

File: src/lesson_6/annotationstasks/Runner.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+создавать экземпляр этого класса, задавать значения его полям и вызывать метод thisClassInfo (использовать возможности
+пакета reflection).
+*/
+
+
+public class Runner {
+
+    public static void main(String[] args) throws Exception {
+
+        new VersionAnnotationProcessor().process(MyService.class);
```
[the remark this answers is not in the document]
Comment (michaelkotor) [reply src/lesson_6/annotationstasks/Runner.java:22]: So, main should contain something like `NameOfApplication.run()`

File: src/lesson_6/maintasks/Task28.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        System.out.println(Arrays.toString(stringArray));
+
+        PolymorphArray<String> polymorphArray2 = new PolymorphArray<>();
+        polymorphArray2.setItem(stringArray);
+
+        System.out.println(polymorphArray2.get(5));
+    }
+}
+
+class PolymorphArray<T> {
```
Comment (michaelkotor) [inline src/lesson_6/maintasks/Task28.java:41]: to extend, you may make a interface and implement this like List and ArrayList

**Review by michaelkotor — CHANGES_REQUESTED** [review]

File: src/lesson_6/maintasks/Task28.java (hunk 2 above)
Comment (EvgeniyTsarkov) [reply src/lesson_6/maintasks/Task28.java]: Done! refactored and tested.

File: src/lesson_6/annotationstasks/Runner.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (EvgeniyTsarkov) [reply src/lesson_6/annotationstasks/Runner.java]: Something's done... not sure exactly if it was what you've asked for. At least, I suggeested some kind of a solution. To the best of my un derstanding.

**Review by michaelkotor — APPROVED** [review]

File: src/lesson_6/annotationstasks/Runner.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (michaelkotor) [reply src/lesson_6/annotationstasks/Runner.java]: Good job!

**Pull request merged**